.travis.yml

```diff
@@ -120,6 +120,7 @@ jobs:
         - USE_MAVEN_REPO="true"
 
 script:
+  - shellcheck ./.ci/*.sh
```

Comment:
make separate job for this.


```
      env:
        - DESC="Shellcheck"
        - CMD="shellcheck ./.ci/*.sh"
        - USE_MAVEN_REPO="false"
```

Reply:
The suppression file is removed temporarily so that we can confirm the SHELLCHECK execution by CI. Once, CI will fail, I will add the suppression file again.

Reply:
@romani Why TRAVIS is not running in the PR. https://app.travis-ci.com/github/checkstyle/checkstyle/requests

Reply:
It might be some problem in config that blocks execution